src/Pact/Gas/Table.hs

```diff
@@ -183,12 +183,21 @@ defaultGasTable =
 
   ]
 
+{-# NOINLINE defaultGasTable #-}
+
 expLengthPenalty :: Integral i => i -> Gas
 expLengthPenalty v = let lv = logBase (100::Float) (fromIntegral v) in 1 + floor (lv^(10::Int))
```

Comment:
Given the finding regarding `Decimal`: was this code path exercised in the performance tests? I think `sort` and `makeList` are the only gas arguments that use this function?

Reply:
I don't think so, no. Since it's `Float` it should be fine though.